Gruntfile.js

```diff
@@ -0,0 +1,34 @@
+module.exports = function (grunt) {
```

Comment:
Another thing I'd really like to do is wire this into something like
grunt-bower-requirejs so we can specify which resources to load on each
page. (Yes, I know, this requires another refactor to _use_ requirejs in
the first place.. just noting that here since this has a pretty substantial
impact on the specifics of implementing that.)

Concatenation is better than nothing but right now we load a huge number of
libraries even when they're not needed and it's adding to the page weight.
Since many events, including Mag events, use "marginal" computers to access
the system, anything we can do to reduce page weight is good.. and the page
weight issue is just going to get worse as we add more libraries to
specific pages for specific functions that aren't needed everywhere in the
application.

On Mon, May 11, 2015 at 1:30 AM, Eli Courtwright <notifications@github.com>
wrote:

> In Gruntfile.js
> <https://github.com/magfest/ubersystem/pull/1105#discussion_r30013729>:
>
> > @@ -0,0 +1,34 @@
> > +module.exports = function (grunt) {
>
> As I explained, this is the Javascript equivalent of a Makefile.
> Currently, we're only doing one thing, which is using the
> grunt-bower-concat library to condense all of our Javascript dependencies
> into a single JS file and a single CSS file.
>
> Theoretically we could minify those files as well if we wanted. I don't
> think that's worth the hassle but it might be worth measuring the
> performance difference at some point just to see. I doubt we'd see any
> perceptible gain, but you never know.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/magfest/ubersystem/pull/1105/files#r30013729>.
>